debezium-core/src/main/java/io/debezium/relational/topic/ByDatabasePlatformTopicMapper.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.relational.topic;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author David Leibovic
+ * @see this doc for useful terminology definitions: https://docs.google.com/document/d/12KZDiv5uUvlB8jnJQJ8KK7PR6whPoWUQEWpX-Ly8n3A/edit#
+ */
+public class ByDatabasePlatformTopicMapper extends TopicMapper {
```

Comment:
I don't intend to actually merge this class into the dbz repo -- it's way to specific to the way we handle sharding at Etsy. I only leave it here in case you are curious how we are implementing a topic mapper. I will remove it before merging.